app/build.gradle

```diff
@@ -57,6 +57,7 @@ dependencies {
         exclude module: 'support-annotations'
     })
 
+    implementation 'com.github.TeamNewPipe:NewPipeExtractor:8114ce6ae4'
```

Comment:
Please remove this line as `dev` is already at a newer commit.

Reply:
I added this line when NewPipe used an older version. I do not think there were any changes in build.gradle after the first merge. This probably just came along due to me merging and rebasing.